app/views/jobseekers/profiles/employments/review.html.slim

```diff
@@ -5,7 +5,7 @@
 
     h1.govuk-heading-l = t(".title")
 
-    = govuk_button_link_to(t(".add_role_button_text"), new_jobseekers_profile_work_history_path)
+    = govuk_button_link_to(t(".add_role_button_text"), new_jobseekers_profile_work_history_path, class: "govuk-button--secondary")
```

Comment:
https://govuk-components.netlify.app/helpers/button/

Think you can just add an option:

`{ secondary: true }`

Reply:
Done, thanks!